EndlessClient/Dialogs/GuildDialog.cs

```diff
@@ -136,7 +150,100 @@ private void SetState(GuildDialogState state)
                         AddItemToList(bankAccountItem, sortList: false);
                     }
                     break;
+
+                case GuildDialogState.Management:
+                    {
+                        ListItemType = ListDialogItem.ListItemStyle.Large;
+                        Buttons = ScrollingListDialogButtons.BackCancel;
+
+                        BackAction += (_, _) => SetState(GuildDialogState.Initial);
```

Comment:
This is going to add the lambda to the `BackAction` event as an additional handler *every* time we change to management state. The impact should be seen if you click management->Back->Management->Back; setting a breakpoint should case it to hit `SetState` twice (although the second will be a no-op since current state == new state).

BackAction should only be set up once in the constructor and should call `SetState` conditionally to the expected prior state based on the current state. You could also push the state onto a stack and pop it off in `BackAction` unconditionally so you don't need to do any additional processing.

Reply:
BackAction only called once now